pkg/ceph/client/pool.go

```diff
@@ -132,7 +171,7 @@ func CreatePool(context *clusterd.Context, clusterName string, newPool CephStora
 func CreatePoolForApp(context *clusterd.Context, clusterName string, newPool CephStoragePoolDetails, appName string) error {
 	// create a crush rule for a replicated pool, if a failure domain is specified
 	replicated := newPool.ErasureCodeProfile == "" && newPool.Size > 0
-	ruleName := fmt.Sprintf("%s-rule", newPool.Name)
+	ruleName := newPool.Name
```

Comment:
So the ruleName the same as the poolname?

Reply:
yes, it's the same, no need to be different. It is found in a different place in the crush schema